### DISCUSSION TITLE
Artifactory support for binary if different os plaform

### DISCUSSION THREAD

**Discussion by riteshnanda09** [goreleaser/goreleaser#1713] [Q&A] [answered]
Guys ,

Have a question around Artifactory , but before that thanks for this tool its amazing simple and yet powerful.

In my case i use goreleaser to generate packages using nfpm for deb and rpm using 

```yaml
nfpms:
--
  | -
  | formats:
  | - deb
  | - rpm

```
i also want to use artifactory feature to upload those as well,  but in our case we have different url's for deb and rpm repos example 

`https://artifactoryserver/repo.deb/pool/`
`https://artifactoryserver/repo.rpm/`

is it possible to do that , i think it should be a common problem for many people if someone has done then how, or if i am doing something wrong with how my repos are setup.

**Comment by caarlos0** [comment, marked as answer]
have you tried with templates on the target url? I think it should be possible 🤔

**Reply by zbindenren** [reply]
@caarlos0 could you elaborate a little?

I have also have
```yaml
formats:
  - deb
  - rpm
```

Right now deb, zip and rpms are uploaded to the same artifactory url.

I would like for example to upload debian packages to:
https://example.com/artifactory/apt/pool/staging/

and rpms to:
https://example.com/artifactory/linux-rhel7-staging/

**Reply by caarlos0** [reply]
ah got it, I think its not possible right now, please open an issue for it.

Thanks!
